src/Plaid/Entity/User.cs

```diff
@@ -17,5 +17,70 @@ public class User
 		[JsonPropertyName("client_user_id")]
 		public string ClientUserId { get; init; } = null!;
+
+		/// <summary>
+		/// The user's full legal name. This is an optional field used in the returning user experience to associate Items to the user.
+		/// </summary>
+		/// <example>Stuart Turner</example>
+		[JsonPropertyName("legal_name")]
+		public string? LegalName { get; set; }
+
+		/// <summary>
+		/// The user's phone number in E.164 format. This field is optional, but required to enable the <a href="https://plaid.com/docs/link/returning-user">returning user</a> experience.
+		/// </summary>
+		/// <example>+14155552671</example>
+		[JsonPropertyName("phone_number")]
+		public string? PhoneNumber { get; set; }
+
+		/// <summary>
+		/// <para>
+		/// The date and time the phone number was verified in ISO 8601 format (yyyy-MM-ddTHH:mm:ssZ). This field is optional, but required to enable any returning user experience.
+		/// </para>
+		/// <para>
+		/// Only pass a verification time for a phone number that you have verified. If you have performed verification but don’t have the time, you may supply a signal value of the start of the UNIX epoch.
+		/// </para>
+		/// <para>
+		/// Example: 2020-01-01T00:00:00Z
+		/// </para>
+		/// </summary>
+		/// <example>2020-01-01T00:00:00Z</example>
+		[JsonPropertyName("phone_number_verified_time")]
+		public string? PhoneNumberVerifiedTime { get; set; }
+
+		/// <summary>
+		/// The user's email address. This field is optional, but required to enable the pre-authenticated returning user flow.
+		/// </summary>
+		/// <example>batman@gothamcity.gov</example>
+		[JsonPropertyName("email_address")]
+		public string? EmailAddress { get; set; }
+
+		/// <summary>
+		/// <para>
+		/// The date and time the email address was verified in ISO 8601 format (YYYY-MM-DDThh:mm:ssZ). This field is optional, but required to enable any returning user experience.
+		/// </para>
+		/// <para>
+		/// Only pass a verification time for an email address that you have verified. If you have performed verification but don’t have the time, you may supply a signal value of the start of the UNIX epoch.
+		/// </para>
+		/// <para>
+		/// Example: 2020-01-01T00:00:00Z
+		/// </para>
+		/// </summary>
+		/// <example>2020-01-01T00:00:00Z</example>
+		[JsonPropertyName("email_address_verified_time")]
+		public string? EmailAddressVerifiedTime { get; set; }
+
+		/// <summary>
+		/// To be provided in the format "ddd-dd-dddd". This field is optional and will support not-yet-implemented functionality for new products.
+		/// </summary>
+		/// <example>18-67-5309</example>
+		[JsonPropertyName("ssn")]
+		public string? Ssn { get; set; }
+
+		/// <summary>
+		/// To be provided in the format "yyyy-mm-dd". This field is optional and will support not-yet-implemented functionality for new products.
+		/// </summary>
+		/// <example>2021-04-22</example>
+		[JsonPropertyName("date_of_birth")]
+		public string? DateOfBirth { get; set; }
```

Comment:
Should be `DateTime?`. The existing `DateTimeConverter` will automatically print in the correct format.